package.json

```diff
@@ -8,8 +8,8 @@
     "lint": "standard && npm run lint:typescript",
     "lint:fix": "standard --fix",
     "lint:typescript": "standard --parser @typescript-eslint/parser --plugin @typescript-eslint/eslint-plugin types/**/*.ts",
-    "test": "tap test/**/*.test.js && tsd",
-    "test:unit": "tap test/**/*.test.js",
+    "test": "ava & tsd",
```

Comment:
It doesn't run the coverage by default.

I assume we need `c8 --100`

Reply:
Good call, I've updated it.